docker-compose.yml

```diff
@@ -3,39 +3,40 @@ services:
     gateway:
         image: gateway:latest
         environment:
-                - PORT=8080
+                - PORT=5000
                 - SECRET_KEY=ineedtoputasecrethere
+                - DATABASE_URL=sqlite:///databasegateway.db
         network_mode: "host"
     bob:
         image: node:latest
         environment:
-                - GRID_NETWORK_URL=http://localhost:8080
+                - GRID_NETWORK_URL=http://localhost:5000
                 - ID=Bob
                 - ADDRESS=http://localhost:3000/
-                - TEST_CONFIG=sqlite:///databaseBob.db
-                - PORT=3000
+                - DATABASE_URL=sqlite:///databaseBob.db
+                - GRID_WS_PORT=3000
```

Comment:
If we keep the PORT variable pattern we will have greater compatibility with different systems/platforms  (Heroku use as default $PORT to setup ports).

Reply:
done